setup.md

```diff
@@ -174,44 +174,161 @@ before pushing them to GitHub.
 
 In order to preview changes locally, you must install the software described below.
 
-1.  **[Ruby](https://www.ruby-lang.org/en/downloads/)**.
+### Windows
 
-    **Linux/macOS**: Ruby is usually included with Linux and macOS ---
-    this is usually sufficient to preview your changes locally.
+#### Option 1 - Using the Windows Subsystem for Linux (WSL)
```

Comment:
Could we please put the different operating system instructions in dropdowns like here: https://datacarpentry.org/genomics-workshop/setup.html